GoogleSignIn/Sources/Public/GoogleSignIn/GIDVerifyAccountDetail.h

```diff
@@ -78,15 +84,15 @@ NS_ASSUME_NONNULL_BEGIN
 ///     address, to be prefilled if possible.
```

Comment:
The doc description above for `presentingViewController` references iOS versions 9 and 10. The minimum iOS version for GSI is 11. No need for the "`SFSafariViewController` on iOS 9 and 10" part. Maybe just: "The view controller used to present the flow."

Reply:
Gotcha, done.